sdk/src/configs/chains.ts

```diff
@@ -1,27 +1,63 @@
-import { arbitrum, avalanche, avalancheFuji, Chain } from "viem/chains";
+import { arbitrum, arbitrumSepolia, avalanche, avalancheFuji, Chain } from "viem/chains";
+
+import { GasLimitsConfig } from "types/fees";
 
 export const AVALANCHE = 43114;
 export const AVALANCHE_FUJI = 43113;
 export const ARBITRUM = 42161;
 export const BSС_MAINNET = 56;
 export const BSС_TESTNET = 97;
 export const ETH_MAINNET = 1;
-
-export const SUPPORTED_CHAIN_IDS = [ARBITRUM, AVALANCHE];
-export const SUPPORTED_CHAIN_IDS_DEV = [...SUPPORTED_CHAIN_IDS, AVALANCHE_FUJI];
-
-export const CHAIN_NAMES_MAP = {
-  [BSС_MAINNET]: "BSC",
-  [BSС_TESTNET]: "BSC Testnet",
+export const BASE_MAINNET = 8453;
```

Comment:
Let's separate settlement and source chains visually and maybe with some comment?